src/main/java/net/rpgtoolkit/common/assets/TileSet.java

```diff
@@ -25,8 +25,6 @@
 
   private LinkedList<Tile> tiles;
 
-  private boolean hasChanged = false;
-
   private int tilesetVersion;
   private int numberOfTiles;
   private int tilesetType;

```

Comment:
This may be out of scope for this PR, but should we just use LinkedLists.size() instead of tracking numberOfTiles separately?